src/pages/blog/starlingx-release-4.md

```diff
@@ -0,0 +1,35 @@
+---
+templateKey: blog-post
+title: StarlingX 4.0 is here!
+author: Ildiko Vancsa
+date: 2020-08-03T01:32:05.627Z
+category: 
+  - label: News & Announcements
+    id: category-A7fnZYrE1
+---
+
+The StarlingX community has just launched the fourth release of the project with various new features and enhancements to the platform.<!-- more -->
+
+The project is approaching a half-year release cycle and this article will give you some insights about what the contributors were working on during the first half of this year.
+
+As the StarlingX platform is fine tuned for edge computing and IoT use cases there are several requirements it has to fulfill in areas such as performance, robustness and security which were in focus during the 4.0 release cycle as well. This blog post will give you insights to some of the new features and components that got integrated into the platform.
+
+Security and proper isolation are crucial in an edge environment as the hardware and software components are located outside of the safety of traditional data centers. This can put challenges on traditional container runtimes as they are sharing the host kernel and use namespaces for separation which can be problematic in case one of the containers gets compromised. There are multiple solutions to this one of which is to use [Kata Containers](https://katacontainers.io) which is a container runtime that provides a fusion between virtual machines and containers. You can now use this project in StarlingX as a runtime that is run by Kubernetes.
+
+To go a step further the StarlingX community has also enabled [Time Sensitive Networking(TSN)](https://1.ieee802.org/tsn/#Published_TSN_Standards) for workloads running in Kata Containers. This is an important functionality as it provides support for determinism in delivering time-sensitive traffic with low and bounded latency while normal traffic is being carried through the same network as well. This is used by applications in industrial IOT, video delivery and other ultra-low latency use cases. As the functionality has dependencies on newer versions of the Linux kernel you will need to perform some configuration steps that the [relevant documentation](https://docs.starlingx.io/developer_resources/stx_tsn_in_kata.html) can guide you through.
+
+Security is also crucial when you manage your hardware infrastructure for instance when you boot or restart the servers that are part of your deployment. Redfish provides well-known standards in this area to provide solutions to this challenge. StarlingX now supports the Redfish Virtual Media Controller that provides you with the option to use a secure BMC-based ISO image boot process.
```

Comment:
Security is also crucial when you manage your hardware infrastructure, for instance, when you boot or restart the servers that are part of your deployment. Redfish provides well-known standards in this area to provide solutions to this challenge. StarlingX now supports the Redfish Virtual Media Controller that provides you with the option to use a secure BMC-based ISO image boot process.